<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetCameraSetting</class>
 <widget class="QWidget" name="WidgetCameraSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Item</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Cameral</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customComboBox" name="comboBox_type"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>x[m]:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_x">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>y[m]:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_y">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-9999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>z[m]:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_z">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Eulerian angle</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>roll[°]:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_roll">
          <property name="toolTip">
           <string>range[-180,180]</string>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>pitch[°]:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_pitch">
          <property name="toolTip">
           <string>range[-180,180]</string>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>yaw[°]:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_yaw">
          <property name="toolTip">
           <string>range[-180,180]</string>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
